lib/ae_mdw/db/format.ex

```diff
@@ -448,6 +448,10 @@ defmodule AeMdw.Db.Format do
     |> put_in(["tx", "gas_used"], Contract.gas_used_in_create(contract_pk, tx_rec, block_hash))
   end
 
+  def custom_encode(:oracle_query_tx, tx, _tx_rec, _signed_tx, _block_hash) do
+    update_in(tx, ["tx", "query"], &Base.encode64/1)
```

Comment:
maybe `["tx", "oracle_query"]`?

Reply:
This is the path of the nested maps that should be updated, if I change that I would be changing the return structure that already exists